esp/esp/program/modules/forms/management.py

```diff
@@ -76,7 +76,7 @@ def save_data(self, cls):
         cls.grade_min = self.cleaned_data['min_grade']
         cls.grade_max = self.cleaned_data['max_grade']
         if not cls.hasScheduledSections():
-            cls.duration = Decimal(str(self.cleaned_data['duration']))
```

Comment:
Do you know why whoever originally wrote this code might have included the str? It seems that removing it did not break anything, but I still wonder why it was there in the first place?

Reply:
I believe that was part of my quick fix for the problems I encountered while fixing #2025. However, that was just a bandaid for the real problem, which is fixed by this rounding of all durations.

Reply:
Okay!